e2e/pageobjects/pages/home.ts

```diff
@@ -0,0 +1,123 @@
+import { Locator, Page, expect } from '@playwright/test';
+
+export class Home {
+  header: Locator;
+  appLogo: Locator;
+  browseBth: Locator;
```

Comment:
I guess it should be `browseBtn`?
```suggestion
  browseBtn: Locator;
```

Reply:
yep, typo fixed in this commit 11978d02cc83787b83083086f7556572fadb6037 

report attached after fix
[playwright-report.zip](https://github.com/reactplay/react-play/files/14759388/playwright-report.zip)